kakarot_scripts/utils/kakarot.py

```diff
@@ -386,27 +388,38 @@ async def get_eoa(private_key=None, amount=10) -> Account:
     )
 
 
-async def send_pre_eip155_transaction(starknet_address, signed_tx: bytes):
+async def send_pre_eip155_transaction(
+    evm_address: str,
+    starknet_address: Union[int, str],
+    signed_tx: bytes,
+    kakarot_owner: Account,
```

Comment:
where is this used?

Reply:
nowhere, thought it would be needed for `invoke` but it's an optional argument :)